### PR TITLE
[TASK] Add note to neos project creation

### PR DESCRIPTION
**Pull request by tbal**

### CODE REVIEW

File: bin/create-project.sh [hunk 1]
```diff
@@ -30,6 +30,7 @@ case "$1" in
     ###################################
     "neos")
         execInDir "$CODE_DIR" "docker run --rm -v $(pwd):/app composer/composer:alpine create-project typo3/neos-base-distribution \"$CODE_DIR\""
+        echo "\nNOTE: You probably want to change the WEB_DOCUMENT_ROOT env in your etc/environment.yml to '/app/Web/' and run 'docker-composer up -d app' to populate the change."
```
Comment (josefglatz) [inline bin/create-project.sh]: Thx for PR! There's a typo `app/Web` should be `app/web`

Comment (tbal) [reply bin/create-project.sh]: @jousch Thanks for your feedback. But that's exactly the point why I added this note. In difference to TYPO3, Neos uses an uppercase dir name, which can be overseen easily. "app/**W**eb" is correct. See: http://neos.readthedocs.io/en/stable/GettingStarted/Installation.html#fundamental-installation point 3

**Comment by mblaschke** [comment]
@tbal There is a conflict, please resolve first :)

**Comment by tbal** [comment]
@mblaschke Fixed :)

**Pull request merged**